File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/P. Shape1/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/P

using System;
using static System.Console;

namespace Codeforces.Assuit.Contest1
{
    class ProblemP
    {
        static void Main(string[] args)
        {
            ushort n = ushort.Parse(Console.ReadLine());
            for(int i=n; i >= 1; i--){
                for(int j=0; j<i; j++)
                    Write("*");
                WriteLine();
            }
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/P. Shape1/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/P
#include<iostream>
#include<cstdio>

using namespace std;
 
int main(){
    unsigned short n;
    cin >> n;
    for(int i=n; i >= 1; i--){
        for(int j=0; j<i; j++)
            cout << "*";
        cout << endl;
    }
}